app/assets/v2/js/grants/_new.js

```diff
@@ -396,7 +400,8 @@ const grant_chains = [
 
 if (document.contxt.is_staff) {
```

Comment:
Should Cosmos be added to the default blockchain dropdown and not just the staff one?

Reply:
@schultztimothy it's usually in staff until rev team gives the go ahead to make it public

Reply:
@schultztimothy it's usually in staff until rev team gives the go ahead to make it public